docs/admininstrator-guide/administrator-slash-commands.md

```diff
@@ -1,2 +1,27 @@
 # Administrator Slash Commands
 
+Administrator slash commands are used to perform system-level functions that require administrator access.
+
+## Install Jira instances
+
+* `/jira instance install cloud [jiraURL]` - Connect Mattermost to a Jira Cloud instance located at `<jiraURL>`
+* `/jira instance install server [jiraURL]` - Connect Mattermost to a Jira Server or Data Center instance located at `<jiraURL>`
+
+## Uninstall Jira instances
+
+* `/jira instance uninstall cloud [jiraURL]` - Disconnect Mattermost from a Jira Cloud instance located at `<jiraURL>`
+* `/jira instance uninstall server [jiraURL]` - Disconnect Mattermost from a Jira Server or Data Center instance located at `<jiraURL>`
+
+## Manage channel subscriptions
+
+* `/jira subscribe` - Configure the Jira notifications sent to this channel
+* `/jira subscribe list` - Display all the the subscription rules setup across all the channels and teams on your Mattermost instance
+
+## Other
+
+* `/jira instance alias [URL] [alias-name]` - Assign an alias to an instance
```

Comment:
Suggest adding the following command

* `/jira instance list` - List installed Jira instances